devel/simple-web-server/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.0
```

Comment:
Please change to and test with the cmake 1.1 portgroup. 

See https://trac.macports.org/ticket/47197 for more information.

Reply:
done